adminguide.rst

```diff
@@ -320,6 +322,52 @@ job queue and record of past jobs.
     If this happens it is best to stop Flux, remove the content.sqlite file,
     and restart.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+flux-accounting configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to support fairshare and multi-factor priority ordering of user jobs, the flux-accounting package must be present and configured for the system instance. This allows for the management of users and banks on a cluster, job priorities to be calculated using multiple factors and uses job record information from users to calculate and maintain fairshare values.
```

Comment:
Also, would be good to end lines at 80 columns rather than wrapping,  for consistency with the rest of the document, and to be better able to review changes in diff form (diff is line oriented).